bundles/org.openhab.binding.knx/README.md

```diff
@@ -235,18 +235,21 @@ For UoM support see the explanations of the `number` channel.
 #### Group Address Notation
 
 ```text
-<config>="[<dpt>:][<]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
+<config>="[<dpt>:][<>]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
 ```
 
 where parts in brackets `[]` denote optional information.
 
+**Each configuration parameter has a `mainGA` where commands are written to and optionally several `listeningGA`s.**
+
+`mainGA` also listens to incoming packets, unless prefixed with a `>` character.
+This is recommended if you have dedicated status group addresses which are added as `listeningGA`s.
+
 The optional `<` sign tells whether the group address of the datapoint accepts read requests on the KNX bus (it does, if the sign is there).
-All group addresses marked with `<` are read by openHAB during startup.
+Group addresses marked with `<` are read by openHAB during startup, but typically this is useful for only one group address per definition.
```

Comment:
```suggestion
The group addresses marked with `<` are read by openHAB during startup. 
Future versions might support reading from one GA only.
```

Or can you imagine a setup where reading several addresses for one state would be correct? The result would be non-deterministic, because the final state would depend on the device with the slowest response.

Reply:
No, I can't. The patch already introduces a warning on info level. The question is how to notify users who use the GUI.

Reply:
I agree here.

>  The question is how to notify users who use the GUI.

That's a good question in many cases ... I can imagine to use the log WebSocket to register to the logger of the add-on of the Thing you currently edit and show some alert in the UI if a warning or error message is received over the WebSocket. But this is just a general idea for the future ... in the mean time, I would say, it's best to have a look at your logs when changing configuration. (I always watch my logs when changing configuration.)